ftw/simplelayout/browser/client/web/js/app/simplelayout/Block.js

```diff
@@ -49,6 +49,13 @@ define([
       return this;
     };
 
+    this.markLinks = function() {
+      var url = window.location.protocol + "//" + window.location.host;
+      // all http links (without the link-plain class), not within this site
+      $("a[href^=\"http\"]:not(.link-plain):not([href^=\"" + url + "\"])", this.element).attr("target", "_blank");
```

Comment:
Whether links should open in a new window or not is a configuration option in Plone; this change does _always_ open links in new windows, no matter what is configured in the property.

I would prefer if simplelayout does not _implement_ this behavior but only _trigger_ the Plone functions, which know and respect the property. See `mark_special_links.js`.